dbt-cta/hustle/models/0_ctes/inbox_addresses_cte1.sql

```diff
@@ -0,0 +1,22 @@
+-- CTE:
```

Comment:
what does CTE mean?

Reply:
common table expression. It's just a ~pretentious~ technically accurate way to describe a query that happens before another query but it's all in the same query but you break it up so that the steps are easier for human brains to contemplate 🧠